packages/ui/src/components/Schemas/TextSchema.tsx

```diff
@@ -5,13 +5,15 @@ import {
   DEFAULT_LINE_HEIGHT,
   DEFAULT_CHARACTER_SPACING,
   DEFAULT_FONT_COLOR,
+  DEFAULT_PT_TO_PX_RATIO,
   TextSchema,
-  calculateDynamicFontSize,
+  calculateDynamicFontSize
 } from '@pdfme/common';
 import { SchemaUIProps } from './SchemaUI';
 import { ZOOM } from '../../constants';
 import { FontContext } from '../../contexts';
-
+import * as fontkit from 'fontkit';
+import { Buffer } from 'buffer/';
```

Comment:
```
import { Buffer } from 'buffer/';
```
↓
```
import { Buffer } from 'buffer';
```
then you can remove  // @ts-ignore line:61

Reply:
Hey @hand-dot, 

When making this update, we noticed that just using the standard
`import { Buffer } from 'buffer';` was still making reference to the built-in Buffer made available from the NodeJS API and not the export from the `buffer` package.

**Using import from 'buffer'**
![Screenshot 2023-07-24 at 12 43 01](https://github.com/pdfme/pdfme/assets/45564901/84d0b00f-689c-45c0-adde-6a4c47ed101f)

**Using import from 'buffer/'**
![Screenshot 2023-07-24 at 12 42 37](https://github.com/pdfme/pdfme/assets/45564901/8326ef35-f5f7-4ed0-86d3-a39754bca11a)

I could be wrong, but wanted to check first

Reply:
@steffancarrington 
Oh... You are right!!

https://github.com/feross/buffer#usage
![Uploading CleanShot 2023-07-25 at 10.40.19.png…]()

Reply:
Hey @hand-dot are we happy to keep this as it is?